packages/xo-web/src/xo-app/user/otp.js

```diff
@@ -0,0 +1,69 @@
+import authenticator from 'otplib/authenticator'
+import Component from 'base-component'
+import crypto from 'crypto'
+import qrcode from 'qrcode'
+import React from 'react'
+import { addOtp, removeOtp } from 'xo'
+
+import { Container, Row, Col } from '../../common/grid'
+import { Toggle } from '../../common/form'
+
+authenticator.options = { crypto }
+
+export default class Otp extends Component {
```

Comment:
Please rewrite this component using Reaclette, keep me posted if you need help :slightly_smiling_face: